src/components/developerCard.tsx

```diff
@@ -0,0 +1,263 @@
+    box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.3);
+    border-radius: 10px;
+  }
+  %::-webkit-scrollbar-thumb {
+    border-radius: 10px;
+    box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.5);
+  }
+`
+
+const InfoSpace = styled.div`
+  display: flex;
+  overflow: hidden;
+  scrollbar-width: thin;
+  padding: 0.2rem;
+
+  &:hover {
+    overflow: auto;
+  }
+  &::-webkit-scrollbar {
+    height: 4px;
+    width: 4px;
+  }
+
+  &::-webkit-scrollbar-track {
+    box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.3);
+    border-radius: 10px;
+  }
+  %::-webkit-scrollbar-thumb {
+    border-radius: 10px;
+    box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.5);
+  }
+`
+const BadgeSkill = styled.div`
+  height: 20px;
+  background-color: #343a40;
+  display: inline-block;
+  font-size: 75%;
+  font-weight: 700;
+  line-height: 1;
+  text-align: center;
+  white-space: nowrap;
+  vertical-align: baseline;
+  border-radius: 10rem;
+  color: #fff;
+  padding: 6px;
+  margin-right: 2px;
+`
+const Skill = styled.span``
+
+const SummaryDiv = styled.div`
+  height: 100px;
+  display: flex;
+  font-size: 75%
+  flex-direction: column;
+  justify-content: center;
+  overflow: hidden;
+  scrollbar-width: thin;
+  font-family: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono",
+    "Courier New", monospace !important;
+  &:hover {
+    overflow: auto;
+  }
+  &::-webkit-scrollbar {
+    height: 4px;
+    width: 4px;
+  }
+
+  &::-webkit-scrollbar-track {
+    box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.3);
+    border-radius: 10px;
+  }
+  %::-webkit-scrollbar-thumb {
+    border-radius: 10px;
+    box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.5);
+  }
+`
+const Summary = styled.p`
+font-size: 80%
+  font-family: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono",
+    "Courier New", monospace !important;
+`
+
+const SocialNetworks = styled.div`
+  font-size: large;
+`
+
+const Media = styled.a`
+  color: #343a40;
+  margin-right: 3px;
+`
+
+const DeveloperCard: React.FC<Props> = ({
+  developerName,
+  initials,
+  image,
+  skillsList,
+  summaryDeveloper,
+  webPage,
+  github,
+  twitter,
+  linkedin,
+}) => {
+  return (
+    <CardDiv>
+      <PhotoDiv>
+        <Photo src={image} />
+      </PhotoDiv>
+      <CardContent>
+        <NameRow>
+          <CicleContainer>
+            <InitialName>{initials}</InitialName>
+          </CicleContainer>
+          <Name>{developerName}</Name>
+        </NameRow>
+
+        <InfoSkill>
+          <InfoSpace>
+            {skillsList.map(skill => (
+              <BadgeSkill key={skill}>
+                <Skill>{skill}</Skill>
+              </BadgeSkill>
+            ))}
+          </InfoSpace>
+        </InfoSkill>
+
+        <SummaryDiv>
+          <Summary>
+            <small>{summaryDeveloper}</small>
+          </Summary>
+        </SummaryDiv>
+        <SocialNetworks>
+          {webPage != null && (
+            <Media href={webPage} target="_blank">
+              <i className="fas fa-globe-americas"></i>
```

Comment:
@ppv95  podrias usar la libreria de[ fontawasome para react](https://fontawesome.com/how-to-use/on-the-web/using-with/react)